documentation/writing_plans.md

```diff
@@ -822,11 +822,22 @@ to the log file.
 
 ### Puppet log functions
 
-To generate log messages from a plan, use the Puppet log function that corresponds to the level you
-want to track: `error`, `warn`, `notice`, `info`, or `debug`. Configure the log level for both log
-files and console logging in `bolt-project.yaml`.  The default log level for the console is `warn`.
-For log files, the default is `notice`.  Use the `--log-level debug` flag to set the console log
-level to `debug` for a single run.
+To generate log messages from a plan, you can use Puppet's built-in log functions and configure
+your Bolt log level accordingly. Be aware that Bolt logs messages at slightly different levels than
+Puppet. Puppet log levels map to Bolt log levels in the following way:
+
+| Puppet log level | Bolt log level |
+| --- | --- |
+| `debug` | `trace` |
+| `info` | `debug` |
+| `notice` | `info` |
+| `warning` | `warn` |
+| `err` | `error` |
+
+Configure the log level for both log files and console logging in `bolt-project.yaml`. The default
+log level for both the console and log files is `warn`. Use the `--log-level debug` flag to set
```

Comment:
Suggesting users set the log level to debug in this section to surface puppet log level debug seems confusing (and  wrong). We show in the table how we "demote" debug to trace in puppet. So if users want to see *puppet* debug level logs in the console they would need to set the log level to trace.

Reply:
@hestonhoffman @barriserloth

Reply:
Okay, let me take another stab at this. I'll open a separate PR and ping both of you

Reply:
Ah, ok. That was part of the docs already, and so I didn't mess with it